guacamole/src/main/webapp/translations/fr.json

```diff
@@ -202,6 +220,7 @@
         "TABLE_HEADER_HISTORY_USERNAME" : "Identifiant",
         "TABLE_HEADER_HISTORY_START"    : "Ouverture",
         "TABLE_HEADER_HISTORY_DURATION" : "Durée",
+        "TABLE_HEADER_HISTORY_REMOTEHOST" : "Ordinateur distant",
```

Comment:
I'd be in favor of lining all of these `TABLE_HEADER_HISTORY` lines up so that the ":" is in the same place.  See other places in the file.